### PR TITLE
Ft pm2 configuration 5

### LINKED ISSUE #5: Feature: Setup PM2 configuration
setup for scalable deployment

### PR DESCRIPTION
**Pull request by walston**
Resolves #5

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -15,8 +16,8 @@ app.use(layout());
 
 // Compile app's static assets.
 app.use(favicon('./public/images/favicon.ico'));
-app.use(serveStatic('./public/images', { 'maxAge': '1y' }));
-app.use(serveStatic('./public', { 'maxAge': '1y' }));
+app.use(express.static('./public/images', { 'maxAge': '1y' }));
+app.use(express.static('./public', { 'maxAge': '1y', }));
```
Comment (samjwalston) [inline app/app.js]: Why is there a comma after '1y'?

File: README.md [hunk 2]
```diff
@@ -24,4 +25,4 @@ Copyright (c) 2016, Sam Walston <sam@monstervr.com>
...
 
-THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
\ No newline at end of file
+THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
```
Comment (samjwalston) [inline README.md]: What happened here? They're the same...

File: README.md [hunk 3]
```diff
@@ -11,7 +11,8 @@ Run the `npm install` command in terminal to install all dependencies.
 
 # Launch
 
-Run the `npm start` command in terminal to launch the server.
+Run `npm start` command in terminal to launch the dev server.
+Run `npm run launch` command in terminal to launch a production server.
```
Comment (samjwalston) [inline README.md]: Should update the Procfile to run the new production launch script

File: README.md (hunk 2 above)
Comment (walston) [inline README.md]: I'm not sure what to do about that... You want it in there though, right?

**Pull request merged**